Jenkinsfile

```diff
@@ -9,7 +9,7 @@ stage('Trust') {
 
 import org.feedhenry.Utils
 
-fhBuildNode(['label': 'nodejs']) {
+fhBuildNode(['label': 'nodejs6']) {
```

Comment:
I see the build was failing on master when this was nodejs, any reason, will we have to bump in the future to nodejs8 and nodejs10 ?

Reply:
@austincunningham No current plans to update past Node6.

Reply:
@austincunningham - `nodejs` build node no longer available - `nodejs6` specifies a node6 build node that is available. As mentioned, no current plans to update past node6